docs/src/test/count_outfiles.ok

```diff
@@ -100,25 +100,25 @@ Info: 5, Warn: 1, Error: 4
 Processing ../src/gpl/messages.txt
 Info: 70, Warn: 10, Error: 14
 Processing ../src/grt/messages.txt
-Info: 29, Warn: 42, Error: 93
+Info: 29, Warn: 42, Error: 88
 Processing ../src/gui/messages.txt
 Info: 1, Warn: 33, Error: 64
 Processing ../src/ifp/messages.txt
 Info: 4, Warn: 7, Error: 17
 Processing ../src/mpl/messages.txt
-Info: 12, Warn: 7, Error: 9
+Info: 12, Warn: 7, Error: 8
 Processing ../src/mpl2/messages.txt
 Info: 2, Warn: 6, Error: 21
 Processing ../src/odb/messages.txt
-Info: 64, Warn: 199, Error: 136
+Info: 64, Warn: 199, Error: 137
```

Comment:
It looks like we are still testing against the number of messages across the whole program.  I'm not clear how this is more local.

Reply:
Apologies. This `count_outfiles` test will be removed. I added a prototype 2 new local tests to src/ant, and modified the docs/src/test to be depending on static files instead. Could you have a look?
ant_man_tcl_check checks for man,proc, readme counts
ant_readme_msgs_check checks for readme counts and messages count.